docs/site/Update-generator.md

```diff
@@ -0,0 +1,64 @@
+---
+lang: en
+title: 'Update generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Update-generator.html
+---
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+The `lb4 update` command runs against a LoopBack 4 project and checks
+dependencies against the installed `@loopback/cli`. By default, it checks for
+exact match. Use `--semver` option to check compatibility using semver
+semantics.
+
+```sh
+lb4 update [options]
+```
+
+If the command is run without a LoopBack 4 project, it prints the version of the
+current CLI and versions of LoopBack packages that are released with the
+`@loopback/cli`. This is the same as `lb4 -v`.
+
```

Comment:
Please explain when and why should users run `lb4 update` instead of `npm update`. I'd like this information to be easy to find, not buried in a pull request discussion.